kolibri/plugins/learn/assets/src/views/explore-page/index.vue

```diff
@@ -22,18 +22,20 @@
 
     <span class="visuallyhidden" v-if="subtopics.length">{{ $tr('navigate') }}</span>
 
-    <card-grid v-if="contents.length">
-      <content-grid-item
-        v-for="content in contents"
-        v-show="selectedFilter.value === 'all' || selectedFilter.value === content.kind"
-        :key="content.id"
-        class="card"
-        :title="content.title"
-        :thumbnail="content.thumbnail"
-        :kind="content.kind"
-        :progress="content.progress"
-        :link="genLink(content)"/>
-    </card-grid>
+    <content-card-grid :contents="contents" v-if="contents.length">
+
+      <template scope="content">
```

Comment:
all the "props" defined in a slot are now associated with the `content` object, specified by `scope`. So you don't loop through the contents here. That's being done in carousel (there's logic in place there so that the right number of contents are only ever rendered, etc.).